sdk/eventhub/eventhubs-checkpointstore-table/karma.conf.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (config) {
     files: [
       // Uncomment the cdn link below for the polyfill service to support IE11 missing features
       // Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys
-      // "https://cdn.polyfill.io/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys|always",
+      // "https://cdnjs.cloudflare.com/polyfill/v2/polyfill.js?features=Symbol,Promise,String.prototype.startsWith,String.prototype.endsWith,String.prototype.repeat,String.prototype.includes,Array.prototype.includes,Object.keys|always",
```

Comment:
Thanks for the PR!  We have dropped IE 11 support a while ago.  Not sure how this file didn't get updated to remove all things IE 11 related.  I'd suggest removing all the comments, include this polyfill URL and the one above it about IE missing features.
```suggestion
```

Reply:
@jeremymeng I made the requested changes. Thanks for reviewing :hugs: